minethd.cpp

```diff
@@ -32,7 +32,7 @@
 
 void thd_setaffinity(std::thread::native_handle_type h, uint64_t cpu_id)
 {
-	SetThreadAffinityMask(h, 1ULL << cpu_id);
+	SetThreadAffinityMask((HANDLE)h, 1 << cpu_id);
```

Comment:
In which header is `HANDLE` defined?
Why should `1` be a 32bit integral type. This would get an overflow on systems with 64 processors.